### PR TITLE
Replace `AMIDeleteOn` tag with `DeleteOn` tag

### PR DESCRIPTION
**Pull request by SweetOps**
## What
* Replace AMIDeleteOn tag with DeleteOn tag

## Why
* Remove redundant `AMI` prefix

### CODE REVIEW

**Review by osterman — APPROVED** [review]

File: ami_backup.py [hunk 1]
```diff
@@ -46,6 +46,6 @@ def lambda_handler(event, context):
     ec.create_tags(
```
Comment (s2504s) [inline ami_backup.py:46]: This tag adds to all instances and AMIs
Instance tag with `Key` = `DeleteOn` and `Value` = `2017-09-04` can cause misunderstanding.
Let's add different tags: 
to the instance - `AMIDeleteOn`
to the AMI - `DeleteOn`

**Review by const-bon — CHANGES_REQUESTED** [review]
@SweetOps It is necessary to test it in order to be sure we do not break backup process.

**Comment by SweetOps** [comment]
has been tested and working well

Backup:
screenshot 2017-09-13 11 13 00
screenshot 2017-09-13 11 13 49
Cleanup:
screenshot 2017-09-13 11 21 11

**Review by osterman — DISMISSED** [review]

File: ami_backup.py (hunk 1 above)
Comment (const-bon) [reply ami_backup.py:46]: We should turn off instance tagging. Instance tag `DeleteOn` may cause confusion.
screenshot from 2017-09-14 18-13-27

**Review by const-bon — CHANGES_REQUESTED** [review]
https://github.com/cloudposse/tf_ami_backup/pull/6#discussion_r138923697

**Comment by SweetOps** [comment]
screenshot 2017-09-20 13 09 17
screenshot 2017-09-20 13 08 35

**Comment by SweetOps** [comment]
screenshot 2017-09-20 14 39 10

**Review by const-bon — APPROVED** [review]

**Pull request merged**